shell/src/main/java/tachyon/shell/TFsShell.java

```diff
@@ -932,7 +932,7 @@ public int touch(TachyonURI path) throws IOException {
    */
   public int unpin(TachyonURI path) throws IOException {
     try {
-      mTfs.setPin(mTfs.open(path), false);
+      mTfs.setPin(mTfs.open(path), true);
```

Comment:
@chen-tian Oh I think there is some confusion here, the setPin command's second argument is to pin or unpin (true is to pin, false is to unpin). I think pin's behavior is always recursive.